mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp

```diff
@@ -1525,6 +1533,17 @@ vectorizeAsTensorPackOp(RewriterBase &rewriter, tensor::PackOp packOp,
   (void)status; // prevent unused variable warning on non-assert builds.
   assert(succeeded(status) && "failed to reify result shapes");
 
+  ArrayRef<int64_t> resultTensorShape = packOp.getDestType().getShape();
```

Comment:
I can't comment on the start of the method. Can you update the doc?

Reply:
Could you explain more?

Reply:
I probably missed it in the previous review. The comment is updated, so this is resolved.